website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -35,18 +35,7 @@ agents require that either the [`start_join`](/docs/agent/opts#start_join) or
 agents. If those stipulations are not met, attempting to start the client or snapshot agent will
 result in it immediately shutting down.
```

Comment:
Is it worth including a link to our FAQ page somewhere in here?

Reply:
Yep